task.sh

```diff
@@ -1 +1,22 @@
-#! /bin/bash
+#!/bin/bash
+
+# Read database credentials from environment variables
+DB_USER=${DB_USER}
+DB_PASSWORD=${DB_PASSWORD}
```

Comment:
Using `${DB_PASSWORD}` is unnecessary here. You can simply use `$DB_PASSWORD` to reference the environment variable.